src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.Builder.cs

```diff
@@ -345,6 +345,34 @@ public void AddRange(ImmutableArray<T> items, int length)
                 }
             }
 
+            /// <summary>
+            /// Adds the specified items to the end of the array.
+            /// </summary>
+            /// <param name="items">The items.</param>
```

Comment:
```suggestion
            /// <param name="items">The items to add at the end of the array.</param>
```
Similarly for the methods below.

Reply:
Fixed.